integration/kubernetes/confidential/fixtures/pod-config-secret.yaml.in

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: secret-test-pod-cc
+spec:
+  runtimeClassName: $RUNTIMECLASS
+  containers:
+  - name: busybox
+    image: quay.io/prometheus/busybox:latest
+    imagePullPolicy: Always
+    command:
+    - sh
+    - -c
+    - |
+      env
+      if env | grep -q "SECRET_PASSWORD=unsealed_secret"; then
```

Comment:
Could we try comparing specifically the `$SECRET_PASSWORD` value, rather than just checking the whole `env` for it? In the unlikely case that we end up with trailing characters set during the unsealing e.g. so `SECRET_PASSWORD = "unsealed_secret "` this test wouldn't fail as is if we are just checking `env` for the match?